src/shogun/lib/parameter_observers/ObservedValue.h

```diff
@@ -70,7 +70,7 @@ namespace shogun
 		 * @param value Any-wrapped value of the param
 		 */
 		ObservedValue(
-		    int64_t step, std::string& name, Any value, SG_OBS_VALUE_TYPE type)
+		    int64_t step, std::string name, Any value, SG_OBS_VALUE_TYPE type)
```

Comment:
const?

Reply:
what is up with the SG_OBS_VALUE_TYPE here?

Reply:
SG_OBS_VALUE_TYPE will be removed. I will update this when the other PR goes in.